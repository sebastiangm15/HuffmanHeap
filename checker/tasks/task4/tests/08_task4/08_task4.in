10
2 S2
3 S3
4 S4
1 S1
8 S8
9 S9
10 S10
5 S5
6 S6
7 S7
6
S2
S4
S6
S8
S10
S1
