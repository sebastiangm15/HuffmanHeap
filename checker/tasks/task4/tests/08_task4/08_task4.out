S10S1S2S3S6S7S8S4S5S9
